.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug, triage]
+assignees:
+  - 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+   - type: textarea
+    id: url
+    attributes:
+      label: URL
+      description: Where did you encounter this error?
+      placeholder: Paste the explorer URL
+      value: ""
+    validations:
+      required: true     
+  - type: dropdown
+id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
+      multiple: true
+      options:
+        - Chrome
+        - Safari
+        - Brave
+        - Firefox
+        - Opera
+        - Microsoft Edge
```

I can't imagine what would go in this category, but if the question is required, maybe we should have an "Other"?

```suggestion
        - Microsoft Edge
        - Other
```